adapter-mongodb/src/main/resources/activities/mongodb-crud-basic.yaml

```diff
@@ -30,7 +29,7 @@ bindings:
 
 blocks:
   - tags:
-      phase: schema
+      block: schema
```

Comment:
same here, ...

Reply:
I think this is an issue with my local rebase, will get those pulled back in.